src/environments/environment.prod.ts

```diff
@@ -10,5 +20,9 @@ export const environment = {
     messagingSenderId: '1003263481418'
   },
   googleAnalyticsTrackingId: '',
-  mapsApiKey: 'AIzaSyCG5P0ZrbWcC212QZsEebyyGwj2KRgs700'
+  mapsApiKey: 'AIzaSyCG5P0ZrbWcC212QZsEebyyGwj2KRgs700',
+  serviceWorkerStrategy: 'registerDelay:5000'
+  // serviceWorkerStrategy: 'registerImmediately'
+  // serviceWorkerStrategy: 'registerWhenStable'
+  // serviceWorkerStrategy: getServiceWorkerObservable
```

Comment:
I would prefer to see these commented out examples of using different strategies in the `app.module.ts`.